modules/nf-core/gubbins/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+
+nextflow_process {
+
+    name "Test Process GUBBINS"
+    script "../main.nf"
+    process "GUBBINS"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gubbins"
+
+    test("test-gubbins") {
```

Comment:
The test name `test-gubbins` is obvious so, better to have the name of the test contain details of the test-data being used.

Thoughts ?

Reply:
Good idea. I have updated the names now!